gcd/src/main/java/org/spine3/server/storage/datastore/DatastoreStorageFactory.java

```diff
@@ -38,55 +39,92 @@
  * @author Alexander Litus
  * @author Mikhail Mikhaylov
  */
+@SuppressWarnings("WeakerAccess") // Part of API
 public class DatastoreStorageFactory implements StorageFactory {
 
     private static final int ENTITY_MESSAGE_TYPE_PARAMETER_INDEX = 1;
 
-    private final DatastoreWrapper datastore;
-    private final Options options;
+    private DatastoreWrapper datastore;
+    private final boolean multitenant;
 
     /**
+     * Creates new instance of {@code DatastoreStorageFactory}.
+     * <p>Same as calling {@link #newInstance(Datastore, boolean)} with {@code false} second argument.
+     *
      * @param datastore the {@link Datastore} implementation to use.
-     * @return creates new factory instance.
+     * @return new instance of the {@code DatastoreStorageFactory}.
      * @see DatastoreOptions
      */
     @SuppressWarnings("WeakerAccess") // Part of API
     public static DatastoreStorageFactory newInstance(Datastore datastore) {
-        return new DatastoreStorageFactory(datastore);
+        return new DatastoreStorageFactory(datastore, false);
     }
 
-    /* package */ DatastoreStorageFactory(Datastore datastore) {
-        options = new Options();
-        this.datastore = new DatastoreWrapper(datastore, options);
+    /**
+     * Creates new instance of {@code DatastoreStorageFactory}.
+     *
+     * @param datastore   the {@link Datastore} implementation to use.
+     * @param multitenant shows if storage factory is configured to serve a multitenant application
```

Comment:
If we do not support multitenancy in the way GCD recommends, let's add a `todo` item for that.

As I recall, the multitenant GCD-based apps should use namespaces for `Entities` (and, thus, `Queries`). I don't think we do it.